cmd/provider/main.go

```diff
@@ -102,6 +104,15 @@ func main() {
 		o.Features.Enable(features.EnableAlphaExternalSecretStores)
 		log.Info("Alpha feature enabled", "flag", features.EnableAlphaExternalSecretStores)
 
+		o.ESSOptions = &controller.ESSOptions{}
+		if *essTLSCertsPath != "" {
```

Comment:
This does not come as `nil` if unset, right?

Reply:
yep, it's never nil